tests/sentry/lang/javascript/test_processor.py

```diff
@@ -688,6 +690,7 @@ def relevant_calls(mock, prefix):
         result = fetch_release_archive_for_url(release, dist=None, url="foo")
         assert result is not None
         assert len(relevant_calls(cache_set, "releasefile")) == 0
+        result.close()
```

Comment:
shouldn't all these `.close()` calls be done in a finaliser, e.g. with the request fixture: `request.addfinalizer(result.close)`?

(this comment is more or less generic to all the test fixes in this PR)

Reply:
Likely yes but since the whole thing tears down anyways I'm not super concerned.